flask_app/src/models/comment.py

```diff
@@ -9,14 +11,16 @@ class Comment(Base):
     id = db.Column(db.Integer, primary_key=True)
     content = db.Column(db.String(1000), nullable=False)
     created_at = db.Column(db.DateTime, default=db.func.current_timestamp())
-    external_name = db.Column(db.String(100), nullable=False)
+    external_name = db.Column(db.String(100), nullable=True)
 
     # FK
     posted_by = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
```

Comment:
```suggestion
    posted_by = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=True)
```
Si on aura des personnes externes, ils ne seront pas dans notre base User